content/en/docs/setup/production-environment/index.md

```diff
@@ -0,0 +1,295 @@
+  - *Role-based access control* ([RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)): Lets you assign access to your cluster by allowing specific sets of permissions to authenticated users. Permissions can be assigned for a specific namespace (Role) or across the entire cluster (ClusterRole). Then using RoleBindings and ClusterRoleBindings, those permissions can be attached to particular users.
+  - *Attribute-based access control* ([ABAC](https://kubernetes.io/docs/reference/access-authn-authz/abac/)): Lets you create policies based on resource attributes in the cluster and will allow or deny access based on those attributes. Each line of a policy file identifies versioning properties (apiVersion and kind) and a map of spec properties to match the subject (user or group), resource property, non-resource property (/version or /apis), and readonly. See [Examples](https://kubernetes.io/docs/reference/access-authn-authz/abac/#examples) for details.
+
+As someone setting up authentication and authorization on your production Kubernetes cluster, here are some things to consider:
+
+- *Set the authorization mode*: When the Kubernetes API server
+([kube-apiserver](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/))
+starts, the supported authentication modes must be set using the *--authorization-mode*
+flag. For example, that flag in the *kube-adminserver.yaml* file (in */etc/kubernetes/manifests*)
+could be set to Node,RBAC. This would allow Node and RBAC authorization for authenticated requests.
+- *Create user certificates and role bindings (RBAC)*: If you are using RBAC
+authorization, users can create a CertificateSigningRequest (CSR) that can be
+signed by the cluster CA. Then you can bind Roles and ClusterRoles to each user.
+See [Certificate Signing Requests](https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/)
+for details.
+- *Create policies that combine attributes (ABAC)*: If you are using ABAC
+authorization, you can assign combinations of attributes to form policies to
+authorize selected users or groups to access particular resources (such as a
+pod), namespace, or apiGroup. For more information, see
+[Examples](https://kubernetes.io/docs/reference/access-authn-authz/abac/#examples).
+- *Consider Admission Controllers*: Additional forms of authorization for
+requests that can come in through the API server include
+[Webhook Token Authentication](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#webhook-token-authentication).
+Webhooks and other special authorization types need to be enabled by adding
+[Admission Controllers](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/)
+to the API server.
+
+## Production workload resources
```

Comment:
I'm not sure what this heading means. Imagine I'm a reader who's heard of Deployment and StatefulSet. Am I expecting to read about hardened replacements for those that come with secure defaults? DeploymentThatForcesYouToDeclareRequiredCapabilities, StatefulSetThatRefusesToRunAsRoot etc?

So, no, that's not what the section is about. However, the heading is not clear enough about what you get when you click it in the nav.

Can we reword? Headings are important to get right because people bookmark them.

Reply:
I was trying to keep the sections on this page parallel:

Production considerations
Production control plane
Production worker nodes
Production user management
Production workload resources

I'll change the last one to read: "Set limits on workload resources"